src/main/java/lambdastreams/task1/WordsStartO.java

```diff
@@ -0,0 +1,14 @@
+package lambdastreams.task1;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WordsStartO {
+    public static List<String> getWordsStartO(String text) {
+        List<String> result = Arrays.stream(text
```

Comment:
this variable is redundant - you can simply return this line (Arrays.stream...)

Reply:
Done